app/controllers/dashboard/districts_controller.rb

```diff
@@ -5,22 +5,43 @@ class Dashboard::DistrictsController < AdminController
 
   EXAMPLE_DATA_FILE = "db/data/example_dashboard_data.json"
 
+  def index
+    authorize :dashboard, :view_my_facilities?
+    @districts = FacilityGroup.all.order(:name)
```

Comment:
This should use some sort of policy scope to only show what a user has access to

Reply:
I cargo culted a scope from another controller, I think its good for this first PR. I'll need to work with someone w/ more experience w/ Pundit to make sure we have this nailed down later.